.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  "extends": "airbnb",
+  "plugins": [
```

Comment:
react, jsx-a11y and import are already included in airbnb: https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnb-1